app/backend/src/couchers/migrations/versions/a0d7c8af5402_modify_the_database.py

```diff
@@ -0,0 +1,270 @@
+"""Modify the database
```

Comment:
Migrations should have a descriptive "commit message"

Reply:
Sorry, I missed this one.

Reply:
I removed the migration from here.